### PR TITLE
Attribute table skeleton (part 1)

### PR DESCRIPTION
**Pull request by tomasMizera**
Icon to open `browseDataPanel`

Stackview for layers list and layer attributes

### CODE REVIEW

**Comment by PeterPetrik** [comment]
would be nice to add some screenshots of new UI

**Comment by saberraz** [comment]
Yes please! A screenshot....:)

**Comment by tomasMizera** [comment]
Screenshots are here :) 

Icon to open attribute table:
drawing

And 2 placeholder pages for layers list and attributes list
<table>
  <tr>
    <td>drawing</td>
    <td>drawing</td>
  </tr>
 </table>

File: app/qml/LayerAttributesPanel.qml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.12
+
+Item {
+  id: root
+
+  property string selectedLayer: ""
+  signal backButtonTapped()
```
Comment (PeterPetrik) [inline app/qml/LayerAttributesPanel.qml:8]: backButtonClicked?

File: app/qml/LayerAttributesPanel.qml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    header: PanelHeader {
+      id: attrPageHeader
+      height: InputStyle.rowHeightHeader
+      width: parent.width
+      color: InputStyle.clrPanelMain
+      rowHeight: InputStyle.rowHeightHeader
+      titleText: qsTr("Attributes")
+      
+      onBack: root.backButtonTapped()
```
Comment (PeterPetrik) [inline app/qml/LayerAttributesPanel.qml:23]: backButtonClicked?

File: app/qml/LayersListPanel.qml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.12
+
+Item {
+  id: root
+
+  signal backButtonTapped()
```
Comment (PeterPetrik) [inline app/qml/LayersListPanel.qml:7]: backButtonClicked?

File: app/qml/LayersListPanel.qml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    header: PanelHeader {
+      id: layersPageHeader
+      height: InputStyle.rowHeightHeader
+      width: parent.width
+      color: InputStyle.clrPanelMain
+      rowHeight: InputStyle.rowHeightHeader
+      titleText: qsTr("Layers")
+      
+      onBack: root.backButtonTapped()
```
Comment (PeterPetrik) [inline app/qml/LayersListPanel.qml:22]: backButtonClicked?

File: app/qml/LayersListPanel.qml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+      withBackButton: true
+    }
+    
+    Column {
+      spacing: 10
```
Comment (PeterPetrik) [inline app/qml/LayersListPanel.qml:27]: spacing: 10 * QgsQuick.Utils.dp

File: app/qml/LayerAttributesPanel.qml (hunk 1 above)
Comment (saberraz) [reply app/qml/LayerAttributesPanel.qml:8]: https://uiux.blog/stop-using-the-term-click-in-a-mobile-experience-479b8ed4f567

Comment (PeterPetrik) [reply app/qml/LayerAttributesPanel.qml:8]: well, in that case we would need to rewrite all other signal so it uses the same 

also we use qt signals that uses "click"

Comment (tomasMizera) [reply app/qml/LayerAttributesPanel.qml:8]: Your call guys :)

Comment (saberraz) [reply app/qml/LayerAttributesPanel.qml:8]: Go with Peter's suggestion! I just send random comments and you can ignore tham

File: app/qml/LayerAttributesPanel.qml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+      rowHeight: InputStyle.rowHeightHeader
+      titleText: qsTr("Attributes")
+      
+      onBack: root.backButtonClicked()
+      withBackButton: true
+    }
+
+    Text {
+      anchors.centerIn: parent
+      text: qsTr("List of attributes for layer " + selectedLayer)
```
Comment (PeterPetrik) [inline app/qml/LayerAttributesPanel.qml:29]: `qsTr("List of attributes for layer %1").arg(selectedLayer)`

you cannot translate "selectedLayer" directly

**Pull request merged**